ProcessMaker/processmaker
# FOUR-8231

## Issue & Reproduction Steps
N/A

## Solution
N/A

## How to Test
- Go to admin
- Select edit an user
- Go to groups tab
- Change the groups and SAVE
- Check the security_logs table for a new log "groupMembershipChanged"

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-8231

## Code Review Checklist


ci:deploy
